### PR TITLE
Cross checking available browsers against configured loaders.

### PR DESCRIPTION
**Pull request by GreenGremlin**
Avoids the `Cannot load browser "${browser": it is not registered! Perhaps you are missing some plugin?` errors.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -62,30 +62,44 @@ var DetectBrowsers = function (config, logger) {
...
     config.detectBrowsers = config.detectBrowsers || {};
+    config.plugins = config.plugins || [];
 
     if (config.detectBrowsers.enabled === false) {
         log.info('Detecting browsers is disabled. The browsers of the browsers array are used.');
         return;
     }
 
-    var availableBrowser = getInstalledBrowsers(browsers);
+    const configuredLaunchers = config.plugins.reduce((prev, plugin) => {
```
Comment (4kochi) [inline index.js]: Can we use es5 syntax please. Just to ensure backwards compatibility.

Comment (GreenGremlin) [reply index.js]: Ah, sorry. I caught myself using es6 a couple times, missed this one. It's fixed now.

**Comment by GreenGremlin** [comment]
Once you're able to merge this, would you mind publishing a release?

**Comment by 4kochi** [comment]
Yes, I guess tomorrow. Will let you know. Thanks for your great work.

**Comment by 4kochi** [comment]
That’s fine. Thx

**Comment by 4kochi** [comment]
Released in `v2.3.0`

**Comment by GreenGremlin** [comment]
Thank you!

**Pull request merged**